pegasus/forms/pd_workshop_materials.rb

```diff
@@ -0,0 +1,24 @@
+class PdWorkshopMaterials
+  def self.normalize(data)
+    result = {}
+
+    result[:enrollment_id_i] = required integer data[:enrollment_id_i]
+    result[:workshop_id_i] = required integer data[:workshop_id_i]
+
+    result[:name_s] = required stripped data[:name_s]
+    result[:school_s] = stripped data[:school_s]
+    result[:address_1_s] = required stripped data[:address_1_s]
+    result[:address_2_s] = stripped data[:address_2_s]
+    result[:city_s] = required stripped data[:city_s]
+    result[:state_s] = required stripped data[:state_s]
+    result[:zip_code_s] = required stripped data[:zip_code_s]
+    result[:email_s] = required stripped data[:email_s]
+    result[:phone_number_s] = required stripped data[:phone_number_s]
```

Comment:
True, this probably won't be used internationally, at least for several more years. Still would prefer all integers that you can format in the views/emails later, but at the very least force them to all have the same US format.